src/components/App/ExpressRecorder.tsx

```diff
@@ -127,7 +127,6 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
         if (recordedBlobs.length) {
             this.recordAgain();
         }
-        this.handleStartClick();
```

Comment:
- It's a breaking change for the case when `recordedBlobs` is empty - the method would do nothing.
- Please update the docs in README according to your changes to the functionality,